scarlet/interpolation.py

```diff
@@ -563,6 +565,23 @@ def sinc_interp_inplace(image, h_image, h_target, angle, pad_shape = None):
     return sinc_interp(image, coord_hr, coord_lr, angle=angle)
 
 
+def interpolate_observation(observation, frame, wave_filter = False):
+    """ Interpolates the images in an observation on the grid described by a frame
```

Comment:
please add parameters and returns to the docstring

Reply:
Done